selfservice/flow/login/hook.go

```diff
@@ -190,10 +190,15 @@ func (e *HookExecutor) PostLoginHook(
 			Info("Identity authenticated successfully and was issued an Ory Kratos Session Token.")
 
 		trace.SpanFromContext(r.Context()).AddEvent(events.NewLoginSucceeded(r.Context(), &events.LoginSucceededOpts{
-			SessionID:  s.ID,
-			IdentityID: i.ID, FlowType: string(a.Type), RequestedAAL: string(a.RequestedAAL), IsRefresh: a.Refresh, Method: a.Active.String(),
-			SSOProvider: provider,
+			SessionID:    s.ID,
+			IdentityID:   i.ID,
+			FlowType:     string(a.Type),
+			RequestedAAL: string(a.RequestedAAL),
+			IsRefresh:    a.Refresh,
+			Method:       a.Active.String(),
+			SSOProvider:  provider,
 		}))
+		trace.SpanFromContext(r.Context()).AddEvent(events.NewSessionIssued(r.Context(), string(s.AuthenticatorAssuranceLevel), s.ID, s.IdentityID))
```

Comment:
Is this the only place? How about https://github.com/ory/kratos/blob/ae22c7cdb58e2b9bd2d147bf890f95fc85b9aef2/selfservice/hook/session_issuer.go#L55?

Should we maybe rather instrument https://github.com/ory/kratos/blob/ae22c7cdb58e2b9bd2d147bf890f95fc85b9aef2/persistence/sql/persister_session.go#L200 ?

Reply:
Added the post registration hook.